repair/row_level.cc

```diff
@@ -1464,7 +1546,7 @@ class repair_meta {
                         } else {
                             last_mf = mf;
                             // On repair follower node, only decorated_key_with_hash and the mutation_fragment inside repair_row are used.
-                            row_list.push_back(repair_row({}, {}, dk_ptr, {}, std::move(mf)));
+                            row_list.push_back(repair_row(std::move(fmf), {}, dk_ptr, {}, std::move(mf)));
```

Comment:
What is this change, and is it related to this patch?

Reply:
Yes,  we need frozen mutation fragment  to get the size of the repair_row so we can throttle.